main.go

```diff
@@ -108,3 +99,15 @@ func main() {
 		return
 	}
 }
+
+func startKafka(srv server.Server, zeroLogger zerolog.Logger) {
+	zeroLogger.Trace().Msg("Initializing Kafka consumers")
+	err := kafka.StartConsumers(&srv, &zeroLogger)
+
+	if err != nil {
+		zeroLogger.Error().Err(err).Msg("Failed to initialize Kafka consumers")
+		// If err is something then start consumer again
+		time.Sleep(10 * time.Second)
+		startKafka(srv, zeroLogger)
```

Comment:
unlikely for us to hit recursion limit i suppose?

Reply:
I think yes. If we hit it then the top level will panic and we'll start again.